.github/workflows/maya-usd-new-issues.yml

```diff
@@ -7,7 +7,9 @@ jobs:
   move-triage-card:
     runs-on: ubuntu-latest
     steps:
-      - uses: alex-page/github-project-automation-plus@v0.3.0
+      - uses: alex-page/github-project-automation-plus@v0.8.3
+      # Aug 2023: Update from v0.3.0 to v0.8.3 which uses node16.
+      #           node12 is out of support.
```

Comment:
How can I test this one?

Reply:
I'm not sure how to test this. I think it is intended to but new issues into the Needs Triage state, but I've never seen it in action.